orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/OrchestrationMigrationAgent.kt

```diff
@@ -65,10 +67,15 @@ class OrchestrationMigrationAgent(
         .toBlocking()
         .single()
 
+      log.info("Found ${unmigratedOrchestrations.size} pipelines to migrate")
+
       if (unmigratedOrchestrations.isNotEmpty()) {
         log.info("${unmigratedOrchestrations.size} orchestrations to migrate ($applicationName) [$index/${allApplications.size}]")
 
         unmigratedOrchestrations.forEach {
+          if (log.isDebugEnabled) {
```

Comment:
i assume you are adding this if check for performance reasons, in which case I suggest caching the value of `log.isDebugEnabled` outside the loop (same comment for https://github.com/spinnaker/orca/pull/3626/files#diff-d6c4e5ff40be45466b0742096fd3fe0cR67)